docs/source/getting_started/1_pycharm.rst

```diff
@@ -1,4 +1,3 @@
-
 PyCharm Professional
```

Comment:
There is no mention of Jaeger. VSCode has "Deploy Jaeger service", Why is there no chapter in PyCharm section?